podcasts/End of Year/Views/StoryIndicator.swift

```diff
@@ -11,7 +13,7 @@ struct StoryIndicator: View {
                         .cornerRadius(Constants.storyIndicatorBorderRadius)
 
                     Rectangle()
-                        .frame(width: geometry.size.width * progress, height: nil, alignment: .leading)
+                        .frame(width: geometry.size.width * (min(max((CGFloat(model.progress) - CGFloat(index)), 0.0), 1.0)), height: nil, alignment: .leading)
```

Comment:
Nit: I found out recently we have a clamp function available: 


```suggestion
                        .frame(width: geometry.size.width * (model.progress - CGFloat(index)).clamped(to: 0.0 ..< 1.0), height: nil, alignment: .leading)
```

Reply:
Done in 9b7351efb7e6d615df4f0d303a0a2c1ecf388b9a